### PR TITLE
Remove dmp-queue pallet from Rococo Asset Hub and Bridge Hub

### PR DESCRIPTION
**Pull request by liamaharon**
DMP queue migration is complete and the pallet should be removed to fix the check runtime upgrade CI.

Screenshot 2023-11-24 at 16 05 37

Screenshot 2023-11-24 at 16 06 44

### CODE REVIEW

**Review by franciscoaguirre — APPROVED** [review]

File: cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml [hunk 1]
```diff
@@ -7,74 +7,86 @@ description = "Rococo variant of Asset Hub parachain runtime"
 license = "Apache-2.0"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive", "max-encoded-len"] }
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [
+	"derive",
+	"max-encoded-len",
```
Comment (ggwpez) [inline cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml:12]: What is this style 🤔 ?

**Review by ggwpez — APPROVED** [review]
Not sure about the toml formatting but the runtime changes LGTM.

File: cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml (hunk 1 above)
Comment (liamaharon) [reply cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml:12]: Default settings of this linter https://taplo.tamasfe.dev/

Comment (liamaharon) [reply cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml:12]: Can turn off the multi-line wrapping if you prefer :P

Comment (ggwpez) [reply cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml:12]: Yea, i think we should keep the crates consistent.

**Pull request merged**